app/scripts/models/reliers/oauth.js

```diff
@@ -136,6 +159,7 @@ define([
       self.importSearchParam('state');
       self.importSearchParam('redirect_uri', 'redirectUri');
       self.importSearchParam('redirectTo');
+      self.importSearchParam('verification_redirect', 'verificationRedirect');
```

Comment:
This should be added to the [OAuth server docs](https://github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md#get-v1authorization). As an aside, `keys` should be as well.